cwms-data-api/src/test/resources/cwms/cda/api/timeseriesprofile/ts_profile.json

```diff
@@ -0,0 +1,12 @@
+{
+  "location-id": {
+    "office-id": "SPK",
+    "name": "Test_TSP_Location"
```

Comment:
Use a "real" name, perhaps related to a dam.

Reply:
Replaced with location name I found in the sample DB data I have